README.md

```diff
@@ -1,81 +1,63 @@
+Some configuration items take a list of values. In this cases the values should be separated by space. **If an empty list is desired the variable shouldn't be created.**
 
-### Configuration
+The main structures is as follows:
 
-The ldap-sync expects a configuration file located at `./config.yaml`. The default path can be changed setting the new path to the environment variable `FAMEDLY_LDAP_SYNC_CONFIG`.
+```
+zitadel
+sources
+  ├── ldap
+  └── ukt
+feature_flags
+```
 
-Also, individual configuration items and the whole configuration can be set using environment variables. For example, the following configuration could be set like this:
+The sources are optional. When a source is configured, the sync tool tries to update users in Famedly's Zitadel instance based on the data obtained from the source.
 
-Configuration yaml
-```yaml
-ldap:
-  url: ldap://localhost:1389
-```
+### LDAP
 
-```bash
-FAMEDLY_LDAP_SYNC__LDAP__URL="ldap://localhost:1389"
-```
+Using caching, LDAP source checks for new, updated, and deleted users in the LDAP server.
 
-Note that the environment variable name always starts with the prefix `FAMEDLY_LDAP_SYNC` followed by keys separated by double underscores (`__`).
+### UKT
 
-Some configuration items take a list of values. In this cases the values should be separated by space. **If an empty list is desired the variable shouldn't be created.**
+This a custom endpoint provided by UKT, which gives a list of email addresses of users that should be deleted from Famedly's Zitadel.
 
-## Quirks & Edge Cases
+## Feature Flags
 
-- Changing a user's LDAP id (the attribute from the `user_id` setting)
-  is unsupported, as this is used to identify the user on the Zitadel
-  end.
-- Disabling a user on the LDAP side (with `status`) results in the
-  user being deleted from Zitadel.
-- Providing multiple values for an LDAP attribute is not supported.
-- Zitadel's API is not fully atomic; if a request fails, a user may
-  not be fully created and still not be functional even if the tool is
-  re-used.
-  - In particular, the matrix localpart, the preferred user name, and
-    whether the user has permissions to use Famedly may not be synced.
-- If a user's email or phone number changes, they will only be
-  prompted to verify it if the tool is configured to make users verify
-  them.
-- Changing a user's email also immediately results in a new
-  login/username.
-- If SSO is turned on later, existing users will not be linked.
+Feature flags are optional and can be used to enable or disable certain features.
+
+### Deactivate only feature
+
+When this feature is set, the sync tool will only synchronize updates, and more specifically, the deactivated users.
+**Note:** Keep in mind LDAP is cached and all the changes made on LDAP will be written to the cache as if they where applied. Therefore, only the deactivation changes will be applied to Zitadel but **all the other changes will be lost**.
```

Comment:
We should probably document other than deactivate_only feature flags. Probably in another PR though

Reply:
Those are documented in the sample yaml, to be fair, maybe we should just expand on the docs there.

Reply:
I moved relevant explanation to config.sample.yaml.